lib/pure/strutils.nim

```diff
@@ -273,19 +273,37 @@ func nimIdentNormalize*(s: string): string =
   ##
   ## That means to convert to lower case and remove any '_' on all characters
   ## except first one.
+  ## All `' '` (spaces) are removed.
+  ##
+  ## .. Note:: All ` (backticks) are preserved iff they are present in `s`.
+  ##
+  ## .. Warning:: No checking (e.g. that identifiers cannot start from
+  ##    digits or '_') is performed.
   runnableExamples:
     doAssert nimIdentNormalize("Foo_bar") == "Foobar"
+    doAssert nimIdentNormalize("Foo Bar") == "Foobar"
```

Comment:
This is dubios. The compiler only does it for backticks.

Reply:
ok, agree, changed that.